src/import/load-costume.js

```diff
@@ -15,18 +15,31 @@ const log = require('../util/log');
  */
 const loadCostumeFromAsset = function (costume, costumeAsset, runtime) {
     costume.assetId = costumeAsset.assetId;
-    if (!runtime.renderer) {
+    const renderer = runtime.renderer;
+    if (!renderer) {
         log.error('No rendering module present; cannot load costume: ', costume.name);
         return costume;
     }
     const AssetType = runtime.storage.AssetType;
-    const rotationCenter = [
-        costume.rotationCenterX / costume.bitmapResolution,
-        costume.rotationCenterY / costume.bitmapResolution
-    ];
+    let rotationCenter;
+    if (costume.rotationCenterX && costume.rotationCenterY && costume.bitmapResolution) {
+        rotationCenter = [
+            costume.rotationCenterX / costume.bitmapResolution,
+            costume.rotationCenterY / costume.bitmapResolution
+        ];
+    }
     if (costumeAsset.assetType === AssetType.ImageVector) {
-        costume.skinId = runtime.renderer.createSVGSkin(costumeAsset.decodeText(), rotationCenter);
-        costume.size = runtime.renderer.getSkinSize(costume.skinId);
+        // createSVGSkin does the right thing if rotationCenter isn't provided, so it's okay if it's
+        // undefined here
+        costume.skinId = renderer.createSVGSkin(costumeAsset.decodeText(), rotationCenter);
+        costume.size = renderer.getSkinSize(costume.skinId);
+        // Now we should have a rotationCenter even if we didn't before
+        if (!rotationCenter) {
+            rotationCenter = renderer.getSkinRotationCenter(costume.skinId);
+            costume.rotationCenterX = rotationCenter[0];
+            costume.rotationCenterY = rotationCenter[1];
+        }
```

Comment:
If it's possible to get into this function without a `bitmapResolution`, should we read back the bitmap resolution here too? Alternatively, should we `delete costume.bitmapResolution` before leaving this function? Right now it looks like some costumes will leave this function with a `bitmapResolution` property and some will leave this function without. (Same comment applies to the other case, below.)

Reply:
Isn't bitmap resolution inapplicable for SVG images? That's why it doesn't get read back out here. IMO, it's okay to leave without the bitmapResolution if it wasn't passed in. The renderer and other places in the VM handle this correctly, and the paint editor is doing away with bitmap resolution... Eventually, I want a refactor where we don't do things with bitmapResolution anywhere except in sb2/.sprite2 import...